openff/recharge/esp/storage/_storage.py

```diff
@@ -6,11 +6,10 @@
 import functools
 from collections import defaultdict
 from contextlib import contextmanager
-from typing import TYPE_CHECKING, ContextManager
+from typing import ContextManager
 
-import numpy
-from openff.units import unit, Quantity
-from openff.recharge._pydantic import BaseModel, Field
+from openff.toolkit import Quantity, Molecule
```

Comment:
(not blocking, just curious) - Quantity here is just re-exported by openff-toolkit, right? So it'd be equivalent to do `from openff.units import Quantity`, but here we're saving a line by just nabbing it from openff-toolkit?

Reply:
Correct, functionally equivalent but I'm trying to nudge towards this usage